code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -357,32 +351,36 @@ GLOBAL_LIST_INIT(gaslist_cache, init_gaslist_cache())
 	var/abs_moved_moles = 0
 
 	//GAS TRANSFER
-	for(var/id in sharer_gases - cached_gases) // create gases not in our cache
-		ADD_GAS(id, gases)
+	for(var/new_gas_id in sharer_gases - cached_gases) // create gases not in our cache
+		ADD_GAS(new_gas_id, cached_gases)
 	for(var/id in cached_gases) // transfer gases
-		ASSERT_GAS(id, sharer)
+		ASSERT_GAS(id, sharer_gases)
 
 		var/gas = cached_gases[id]
 		var/sharergas = sharer_gases[id]
 
-		var/delta = QUANTIZE(gas[ARCHIVE] - sharergas[ARCHIVE])/(atmos_adjacent_turfs+1) //the amount of gas that gets moved between the mixtures
+		///the amount of gas that gets moved between the mixtures.
+		var/gas_delta = QUANTIZE(gas[ARCHIVE] - sharergas[ARCHIVE]) / (atmos_adjacent_turfs + 1)
```

Comment:
why rename delta?

Reply:
was more decriptive imo

Reply:
I'm gonna be a boomer but I think the code's cleaner with it just being delta.
Maybe I'm dumb. idk

Reply:
well i think the bikeshed should be red. ill change it back tomorrow

Reply:
renamed